### PR TITLE
ACMS-1830: Use global executable and fallback to local binary for com…

### PR DESCRIPTION
**Pull request by deepakmishra2**
…poser/drush etc.

### CODE REVIEW

**Review by rajeshreeputra — APPROVED** [review]
Changes looks good.

File: src/Helpers/Process/Commands/CommandBase.php [hunk 1]
```diff
@@ -47,6 +48,13 @@ class CommandBase implements CommandInterface {
    */
   protected $input;
 
+  /**
+   * List of common package.
+   *
+   * @var array
+   */
+  protected $package = ['composer', 'drush'];
```
Comment (vishalkhode1) [inline src/Helpers/Process/Commands/CommandBase.php:56]: @deepakmishra2 What's the purpose of this ? We should not restrict this to drush/composer command. This should be done for all commands.

File: tests/packages_alter.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ acquia/acquia-cms-starterkit:
 
 # Do not install acquia_cms and Lightning modules.
 acquia/acquia_cms: ~
+acquia/blt : ~
```
Comment (vishalkhode1) [inline tests/packages_alter.yml:6]: @deepakmishra2 We can revert as blt released.

**Review by vishalkhode1 — CHANGES_REQUESTED** [review]

File: tests/packages_alter.yml (hunk 2 above)
Comment (apathak18) [reply tests/packages_alter.yml]: Reverted

File: src/Helpers/Process/Commands/CommandBase.php (hunk 1 above)
Comment (apathak18) [reply src/Helpers/Process/Commands/CommandBase.php]: Done the changes

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**